packages/boot/test/orchestration/axelar-gmp.test.ts

```diff
@@ -0,0 +1,445 @@
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { buildVTransferEvent } from '@agoric/orchestration/tools/ibc-mocks.js';
+import { makeTestAddress } from '@agoric/orchestration/tools/make-test-address.js';
+import { BridgeId } from '@agoric/internal';
+import fetchedChainInfo from '@agoric/orchestration/src/fetched-chain-info.js';
+import { defaultAbiCoder } from '@ethersproject/abi';
+import { utils } from 'ethers';
+import { eventLoopIteration } from '@agoric/internal/src/testing-utils.js';
+import type { ExecutionContext, TestFn } from 'ava';
+import { makeWalletFactoryContext } from '../bootstrapTests/walletFactory.js';
+
+type WalletFactoryContext = Awaited<
+  ReturnType<typeof makeWalletFactoryContext>
+>;
+
+export type TestContext = WalletFactoryContext & {
+  wallet: any;
+  previousOfferId: string | null;
+};
+
+const test = anyTest as TestFn<Awaited<ReturnType<typeof makeTestContext>>>;
+
+const makeTestContext = async (t: ExecutionContext) => {
+  const ctx = await makeWalletFactoryContext(
+    t,
+    '@agoric/vm-config/decentral-itest-orchestration-config.json',
+  );
+
+  const wallet =
+    await ctx.walletFactoryDriver.provideSmartWallet('agoric1makeAccount');
+
+  const fullCtx = {
+    ...ctx,
+    wallet,
+    previousOfferId: null,
+  };
+
+  return fullCtx;
+};
+
+let evmTransactionCounter = 0;
+
+/**
+ * @typedef {object} MakeEVMTransactionParams
+ * @property {import('@agoric/smart-wallet/src/smartWallet.js').SmartWallet} wallet
```

Comment:
I think this is `ReturnType<typeof makeWalletDriver>`. Also - you can use `type MakeEVMTransactionParams = { ... }` if you wanted since we're in TS

Reply:
moreover this jsdoc comment is not parsed in `.ts` files.